test/Microsoft.ApplicationInsights.AspNet.Tests/FakeTelemetryChannel.cs

```diff
@@ -6,6 +6,8 @@
     // TODO: Remove FakeTelemetryChannel when we can use a dynamic test isolation framework, like NSubstitute or Moq
```

Comment:
I believe Moq was compiling on both for me. I have limited experience with those. I'd prefer to test on both targets unless it significantly worse.